packages/wonder-blocks-cell/src/components/internal/cell-core.tsx

```diff
@@ -342,7 +342,7 @@ const styles = StyleSheet.create({
     disabled: {
         background: semanticColor.surface.primary,
         borderRadius: theme.root.border.radius.default,
-        color: semanticColor.core.foreground.inverse.subtle,
+        color: theme.root.color.disabled.foreground,
```

Comment:
The disabled colors for TB and Classic are different so I set up theme tokens for the color!